watch-library/shared/watch/watch_utility.c

```diff
@@ -83,6 +83,44 @@ uint8_t is_leap(uint16_t y)
 	return !(y%4) && ((y%100) || !(y%400));
 }
 
+uint8_t get_dst_status(watch_date_time date_time) {
```

Comment:
I think this function as currently written only applies to US DST rules. European DST dates are different: https://greenwichmeantime.com/daylight-saving-time/rules/eu/

This might need a toggle for US or EU DST rules in the settings? Or maybe a fully-flexible setting for the user to input what days/times to apply DST? None of these options are good but DST just sucks :(.